src/EPR.Calculator.Frontend/Controllers/DefaultParametersController.cs

```diff
@@ -72,7 +72,7 @@ public async Task<IActionResult> Index()
                         this.ViewBag.MaterialityThreshold = GetSchemeParametersBasedonCategory(defaultSchemeParameters, ParameterType.MaterialityThreshold);
                         this.ViewBag.BadDebtProvision = GetSchemeParametersBasedonCategory(defaultSchemeParameters, ParameterType.BadDebtProvision);
                         this.ViewBag.TonnageChange = GetSchemeParametersBasedonCategory(defaultSchemeParameters, ParameterType.TonnageChangeThreshold);
-                        this.ViewBag.EffectiveFrom = defaultSchemeParameters.First().EffectiveFrom;
+                        this.ViewBag.EffectiveFrom = defaultSchemeParameters[0].EffectiveFrom;
```

Comment:
I remember @udaydefra asked us to use First() instead of magic numbers. Could you please check with @udaydefra ?

Reply:
I will revert to .First() and add sonarqube nowarn for this.

Reply:
I reverted .First and added <NoWarn> for csharpsquid:S6608